default.cfg.example

```diff
@@ -36,8 +36,20 @@ gaptop = 200
 
 #Daily lend rate threshold after which we offer lends for x days as opposed to 2.
 #If set to 0 all offers will be placed for a 2 day period (0.003-5)
-xdaythreshold = 0.2
-xdays = 60
+vdaythreshold = 0.4
+vdays = 60
+
+wdaythreshold = 0.35
+wdays = 45
+
+xdaythreshold = 0.32
+xdays = 30
+
+ydaythreshold = 0.31
+ydays = 15
+
+zdaythreshold = 0.3
+zdays = 4
```

Comment:
As @rnevet suggested:
Having the settings as paired days to rates in the form of lists, such as:
```
xdaythreshold = 0.1, 0.2, 0.3, 0.4
xdays = 4, 8, 16, 32
```

Is much more favorable than having 5 limited options that clog up the config file. What if I only want 4 limits? 6?
Having a paired list and iterating through it would allow per-person expandability.

If you change it to function as I described, it would be a Yes from me.

If you would rather leave it to me, that's fine too, as I see you have allowed us to make changes.